src/AppBundle/Controller/Admin/LectureController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use JavierEguiluz\Bundle\EasyAdminBundle\Controller\AdminController as BaseAdminController;
+use AppBundle\Entity\Lecture;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Class LectureController
+ * @package AppBundle\Controller\Admin
+ */
+class LectureController extends BaseAdminController
+{
+    /**
+     * {@inheritdoc}
+     *
+     * @return Response|RedirectResponse
+     */
+    public function editLectureAction()
+    {
+        $easyadmin = $this->request->attributes->get('easyadmin');
+        $entity = $easyadmin['item'];
+        $this->denyAccessUnlessGranted('edit', $entity);
+
+        return parent::editAction();
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
@inheritdoc naudojamas tik is extendintos klases overridintiems metodams. Tada PhpStormo, ar koks kitas  phpdoc helperis rodo dokumentacija is to overridinto metodo